diff --git a/node_modules/primefaces/PrimeFaces.d.ts b/node_modules/primefaces/PrimeFaces.d.ts
index 842515b..776067a 100644
--- a/node_modules/primefaces/PrimeFaces.d.ts
+++ b/node_modules/primefaces/PrimeFaces.d.ts
@@ -3,8 +3,10 @@
 /// <reference types="cropperjs" />
 /// <reference types="downloadjs" />
 /// <reference types="google.maps" />
+/// <reference types="inputmask" />
 /// <reference types="jquery" />
 /// <reference types="jqueryui" />
+/// <reference types="js-cookie" />
 /// <reference types="moment-timezone" />
 // Type definitions for PrimeFaces
 // Project: PrimeFaces https://github.com/primefaces
